lib/go-tc/deliveryservices.go

```diff
@@ -49,6 +49,14 @@ const MaxRangeSliceBlockSize = 33554432
 // values of a Delivery Service's 'Active' property (v3.0+).
 type DeliveryServiceActiveState string
 
+// These names of URL query string parameters are not allowed to be in a
+// Delivery Service's "ConsistentHashQueryParams" set, because they collide with
+// query string parameters reserved for use by Traffic Router.
+const (
+	ReservedConsistentHashingQueryParameterFormat = "format"
+	ReservedConsistentHashingQueryParameterTRRED  = "trred"
```

Comment:
It's true that reserved consistent hashing query parameters includes these, but that doesn't describe what they actually are. For reuse, should these be grouped with other Traffic Router-related constants?

For example, this is a header, not a GET variable, but maybe GET variables could go somewhere nearby:

https://github.com/apache/trafficcontrol/blob/7741530c494d797758d226885abee3ed7feda7be/lib/go-tc/traffic_router.go#L55-L57

Reply:
I don't really care where they go, but I also don't think it matters that this doesn't describe what they are to Traffic Router. It describes what they are in the context of Delivery Service properties: disallowed values. Traffic Router doesn't use any part of this Go package, so nobody and nothing reading that code should ever need to understand how Traffic Router uses them.